pkg/controllers/termination/terminate.go

```diff
@@ -65,15 +65,15 @@ func (t *Terminator) drain(ctx context.Context, node *v1.Node) (bool, error) {
 
 	// 2. Separate pods as non-critical and critical
 	// https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
-	for _, pod := range pods {
-		if val := pod.Annotations[v1alpha5.DoNotEvictPodAnnotationKey]; val == "true" {
-			logging.FromContext(ctx).Debugf("Unable to drain node, pod %s has do-not-evict annotation", pod.Name)
+	for _, nodePod := range pods {
+		if val := nodePod.Annotations[v1alpha5.DoNotEvictPodAnnotationKey]; val == "true" {
+			logging.FromContext(ctx).Debugf("Unable to drain node, nodePod %s has do-not-evict annotation", nodePod.Name)
```

Comment:
We don't want to expose the variable name `nodePod` in the log line.

Reply:
I also worry a bit about the amount of logging this will cause. Right now, we don't log on a per pod basis for normal operational behavior. We do log at the pod level for errors. I worry a bit that this will make scale down very noisy. I wonder if we need to introduce a trace level or perhaps only log in some scenarios.

Reply:
Poking around at our debug logs, I think I'm probably fine with this, given the options.

Reply:
I blame Goland, eager to rename var names even though they are just strings :/ Fixed. Also, these log lines were already there, they were not added.